types/ramda/index.d.ts

```diff
@@ -435,7 +431,7 @@ export function curryN<N extends number>(length: N): <F extends (...args: any) =
 export function dec(n: number): number;
 
 /**
- * Returns the second argument if it is not null or undefined. If it is null or undefined, the
+ * Returns the second argument if it is not null or undefined or NaN. If it is null or undefined or NaN, the
```

Comment:
Can we have listing comma? `If it is null, undefined or NaN`

Reply:
Ok, I will change it to "," .

Reply:
How to add NaN as a type ? When I add it after `null | undefined`, Typescript says it is a value, not a type.

Reply:
Done.

Reply:
This is unfortunately not possible. There is [an issue for it](https://github.com/Microsoft/TypeScript/issues/28682) but JS doesn't have a NaN type itself: 
![image](https://user-images.githubusercontent.com/7280931/143870857-f4c46d8b-47c9-419b-8914-91fc8d6a700c.png)

Reply:
ok